src/axis.ts

```diff
@@ -286,6 +285,7 @@ const AXIS_PROPERTIES_INDEX: Flag<keyof Axis> = {
 
 const VG_AXIS_PROPERTIES_INDEX: Flag<keyof VgAxis> = {
   scale: 1,
+  gridScale: 1,
```

Comment:
Not sure why you have to move this down. -- It's causing so many diffs

Reply:
It had to move from AXIS_PROPERTIES_INDEX to VG_AXIS_PROPERTIES_INDEX